src/runtime/Module.cpp

```diff
@@ -368,18 +368,29 @@ Instance* Module::instantiate(ExecutionState& state, const ExternVector& imports
 #if !defined(NDEBUG)
 void ModuleFunction::dumpByteCode()
 {
-    printf("function type %p\n", m_functionType);
-    printf("requiredStackSize %u, requiredStackSizeDueToLocal %u\n", m_requiredStackSize, m_requiredStackSizeDueToLocal);
+    printf("%-34s: %-10p\n", "function type", m_functionType);
+    printf("%-34s: %u bytes\n", "required stack size", m_requiredStackSize);
+    printf("%-34s: %u bytes\n", "required stack size due to local", m_requiredStackSize);
+    printf("%-34s: %zu bytes\n", "pointer size", sizeof(void*));
+    printf("%-34s: %zu bytes", "bytecode size", m_byteCode.size());
+    if (m_byteCode.size() % sizeof(void*) == 0)
```

Comment:
If the total size is divisible by pointer size, it does not mean all items are also start on aligned address. This could be removed.